examples/handlers/number_games.links

```diff
@@ -0,0 +1,65 @@
+# Classic guess a number game
+typename Comp(e::Eff,a) = () ~e~> a;
+typename Answer = [|Low|Correct|High|];
+
```

Comment:
added new example here?

Reply:
Apparently, yes... This is the example from my ProWeb talk.